packages/api/src/dotenv.js

```diff
@@ -0,0 +1,5 @@
+// We need a separate file because of this issue: https://github.com/babel/babel/issues/2061
```

Comment:
Oh booooo. But cool.

Reply:
It's apparently a babel only thing. If we eventually migrate to `tsc` we could avoid it 🌚 

(actually, now that we don't access env vars from static context anymore, I could even revert this change. lmk if you prefer that)